server/rest/src/main/java/org/infinispan/rest/resources/SecurityResource.java

```diff
@@ -80,9 +82,18 @@ public Invocations getInvocations() {
             .invocation().methods(PUT).path("/v2/security/roles/{principal}").withAction("deny")
             .permission(AuthorizationPermission.ADMIN).name("ROLES DENY").auditContext(AuditContext.SERVER)
             .handleWith(this::deny)
+            .invocation().methods(POST).path("/v2/security/cache").withAction("flush")
+            .permission(AuthorizationPermission.ADMIN).name("ACL CACHE FLUSH").auditContext(AuditContext.SERVER)
+            .handleWith(this::aclCacheFlush)
```

Comment:
Not strictly related to this PR, but well, let's not waste the context :)
Could add an indent or a line break between each invocation declaration to make it easier to read?